app/dashboard/migrations/0146_populate_brightid_identifier.py

```diff
@@ -0,0 +1,43 @@
+from django.db import migrations, models, transaction
+import uuid
+
+
+def gen_uuid_0(apps, schema_editor):
+    Profile = apps.get_model('dashboard', 'Profile')
+
+    existing_uuid = []
+
+    profiles_with_uuid = Profile.objects.filter(brightid_uuid__isnull=False)
+    for profile in profiles_with_uuid:
+        brightid_uuid = profile.brightid_uuid
+        print(f"UUID for profile -> {profile.pk} -> {brightid_uuid}")
+        while brightid_uuid in existing_uuid:
+            brightid_uuid = uuid.uuid4()
+            existing_uuid.append(brightid_uuid)
+            profile.save()
+
+
+    while Profile.objects.filter(brightid_uuid__isnull=True).exists():
+        with transaction.atomic():
+
+            for profile in Profile.objects.filter(brightid_uuid__isnull=True)[:1000]:
```

Comment:
hardcoded limit is gonna get us in trtouble in prod where there are 40k profiles

also whats the plan for new profiles, how are they going to get a brightid uuid?

Reply:
A uuid is randomly generated right ? 
So a new user -> would automatically get assigned one which is unique (as defined in the model)
Is my understanding right @apbendi  ?

Reply:
Yes, if I've done this right, a new UUID should be generated when a user is created, per this migration:

https://github.com/gitcoinco/web/pull/7333/files#diff-f2afa72d3c42b2cac75651dd47a6c961R14

Can you sanity check that looks correct.

I'm not sure about the 1K limit @thelostone-mc, why did you have that in there? Did it take 2 minutes to run for only 1K records or for the full 40K?

Reply:
2 min for the whole 40k !